postinstall.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+# this shell script will need wsl on windows, but bina already requires that
+
+bina tdewolff/minify --install-dir node_modules/.bin
+
+# TODO this isn't ideal but i can't find any other way to do it
+cargo install minifier --no-track --root node_modules/.bin/cargo
```

Comment:
Why `--no-track`?

Also, is this setup compatible with renovate for auto updates? https://github.com/renovatebot/renovate

Reply:
https://doc.rust-lang.org/cargo/commands/cargo-install.html

if it's not there cargo will make an unnecessary metadata file, originally i added it with a different directory where that mattered more, but it still has no reason to create the file so i left it

Reply:
Is it different from a lock file?

Reply:
cargo install docs say that it's to keep track of installed packages, this isn't needed here because they never need to be later managed with the cli, all that's needed is the binary file